azerothcore/azerothcore-wotlk
# fix(SQL): dungeon_access_template min_level

## Changes Proposed:
- update min level for each dungeon in dungeon_access_template to use the values from the dbc files



## Issues Addressed:
- Closes 



## SOURCE:

https://wow.tools/dbc/?dbc=lfgdungeons&build=3.3.5.12340#page=1

## Tests Performed:

- apply sql


## How to Test the Changes:

- apply sql
- set `DungeonAccessRequirements.LFGLevelDBCOverride = 1` in your worldserver.conf
- run `.reload dungeon_access_template` & `.reload conf`
- use rdf at different level's, make sure your not placed into dungeons too far above your level.

## Known Issues and TODO List:



## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
